examples/run_largecyc_CPMG.py

```diff
@@ -0,0 +1,202 @@
+"""
+CPMG with large phase cycle
```

Comment:
Unfortunately merging my branch into this branch was a bad move.  We need to finalize my pull request first.

Reply:
the idea I had was merging this into your branch then merging to master but we won't do that anymore

Reply:
but remember -- we need to keep PRs so that they only do *one thing at a time* and are *as simple as possible*. 

Now, we need to wait for the other PR to get merged before we can merge this, or else comments, etc, for the other guy get lost.